### PR TITLE
Execute version check only for SBL update

### PR DESCRIPTION
**Pull request by raghavag**
This patch will run version check only for SBL update and will
skip for all other components.

Version check for all other components will be added in future
patches.

Signed-off-by: Raghava Gudla <raghava.gudla@intel.com>

### CODE REVIEW

File: PayloadPkg/FirmwareUpdate/FirmwareUpdate.c [hunk 1]
```diff
@@ -1130,10 +1130,12 @@ UpdateSystemFirmware (
...
-  Status = VerifyFwVersion (ImageHdr, FwPolicy);
-  if (EFI_ERROR (Status)) {
-    DEBUG ((DEBUG_ERROR, " VerifyFwVersion failed with Status = 0x%x\n", Status));
-    return Status;
+  if (CompareGuid(&ImageHdr->UpdateImageTypeId, &gSblFWUpdateImageFileGuid)) {
+    Status = VerifyFwVersion(ImageHdr, FwPolicy);
+    if (EFI_ERROR (Status)) {
+      DEBUG ((DEBUG_ERROR, " VerifyFwVersion failed with Status = 0x%x\n", Status));
+      return Status;
+    }
```
Comment (gdong1) [inline PayloadPkg/FirmwareUpdate/FirmwareUpdate.c]: VerifyFwVersion () will decide how to check the version, check which components, etc.
So better to update VerifyFwVersion ()  for this change.

Comment (mauricema) [reply PayloadPkg/FirmwareUpdate/FirmwareUpdate.c]: Agree with Guo

Comment (mauricema) [reply PayloadPkg/FirmwareUpdate/FirmwareUpdate.c]: Also CompareGuid() returns INTN.  So you need to use if (CompareGuid() != 0)

Comment (raghavag) [reply PayloadPkg/FirmwareUpdate/FirmwareUpdate.c]: Done, updated code according to the comments

**Pull request merged**